javascript/extractor/lib/typescript/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+load("@//:common.bzl", "on_windows")
+
+# Builds a zip file of the compiled typscript-parser-wrapper and its dependencies.
+genrule(
+    name = "typescript",
+    srcs = [
+        "tsconfig.json",
+        "package.json",
+        "package-lock.json",
+    ] + glob(["src/**"]),
+    outs = ["javascript.zip"],
+    cmd = " && ".join([
```

Comment:
genrules commands are run with `set -euo pipefail`, so you can use `\n` instead of `&&` (which helps when printing failures)

Reply:
Neat, I didn't know!